WooCommerce/Classes/ViewRelated/Dashboard/Settings/Help/ApplicationLogViewController.swift

```diff
@@ -0,0 +1,264 @@
+import UIKit
+import CocoaLumberjack
+
+
+// MARK: - ApplicationLogViewController
+//
+class ApplicationLogViewController: UIViewController {
+
+    /// Main TableView
+    ///
+    @IBOutlet weak var tableView: UITableView!
+
+    /// Table Sections to be rendered
+    ///
+    private var sections = [Section]()
+
+    /// Access the shared DDFileLogger
+    ///
+    let fileLogger = AppDelegate.shared.fileLogger
+
+    /// List of log files
+    ///
+    var logFiles = [DDLogFileInfo]()
+
+    /// Date formatter
+    ///
+    let dateFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateStyle = .medium
+        formatter.doesRelativeDateFormatting = true
+        formatter.timeStyle = .short
+
+        return formatter
+    }()
+
+
+    // MARK: - Overridden Methods
+    //
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigation()
+        configureMainView()
+        configureTableView()
+        loadLogFiles()
+        configureSections()
+        registerTableViewCells()
+    }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if let vc = segue.destination as? ApplicationLogDetailViewController, segue.identifier == Segues.detailSegue {
+            let logFileInfo = sender as! DDLogFileInfo
```

Comment:
Getting the file contents here also seems weird. Is there a better place for this?